site/profile/manifests/core/icinga_agent.pp

```diff
@@ -0,0 +1,99 @@
+# @summary
+#   Icinga agent creation for metric collections
+
+class profile::core::icinga_agent(
+  Boolean $install,
+  String $icinga_master_fqdn,
+  String $icinga_master_ip,
```

Comment:
Why is both the a DNS name and an IP being used to configure the master?  It is better to use only DNS.

Reply:
the requirements showed at https://github.com/Icinga/puppet-icinga2 requires that the agents have explicitly both fqdn and ip (tried without it and icinga2 service brakes)